tests/tests_common_test.go

```diff
@@ -2,6 +2,7 @@ package tests
 
 import (
 	"fmt"
+	"k8s.io/apimachinery/pkg/runtime/schema"
```

Comment:
Moving the import line is not needed.

Reply:
I didn't
I think it was the editor's import tool

Reply:
done @akrejcir